script/check-config.sh

```diff
@@ -164,17 +164,21 @@ echo
 echo 'Generally Necessary:'
 
 echo -n '- '
-cgroupSubsystemDir="$(awk '/[, ](cpu|cpuacct|cpuset|devices|freezer|memory)[, ]/ && $3 == "cgroup" { print $2 }' /proc/mounts | head -n1)"
-cgroupDir="$(dirname "$cgroupSubsystemDir")"
-if [ -d "$cgroupDir/cpu" -o -d "$cgroupDir/cpuacct" -o -d "$cgroupDir/cpuset" -o -d "$cgroupDir/devices" -o -d "$cgroupDir/freezer" -o -d "$cgroupDir/memory" ]; then
-	echo "$(wrap_good 'cgroup hierarchy' 'properly mounted') [$cgroupDir]"
+if [[ $(stat -f -c %t /sys/fs/cgroup 2>/dev/null) = "63677270" ]]; then
```

Comment:
`[[` isn't necessary here; technically we should do `$()` in a separate command to catch the failure, but in this case we don't actually care about the failure (and the `else` should cover it) so it's fine:

```suggestion
if [ "$(stat -f -c %t /sys/fs/cgroup 2>/dev/null)" = '63677270' ]; then
```

Reply:
Thanks, makes sense (I am unsure why I used `[[` in the first place).

Fixed.